test_team_2.py

```diff
@@ -1,7 +1,23 @@
 """Unit test file for Team Team2"""
 import unittest
+import re
+from urllib.parse import urlparse, parse_qs
 from pii_scan import anonymize_text
 
+def check_url(url):
```

Comment:
check_url() should be moved to pii_scan.py and imported similar to anonymize_text()